### ISSUE TITLE
🪲[Bug]: Nodemon Crashes due to mongoose url as can't fetch it.

### ISSUE THREAD

**Issue by shr3012** [sujaltangde/JobLane#195]
There’s an error because Mongoose is expecting a MongoDB connection link, but it’s receiving "undefined." You’ll need to adjust the code in database.js and server.js to correctly pass the connection string.

**Comment by Slacky300** [comment]
> There’s an error because Mongoose is expecting a MongoDB connection link, but it’s receiving "undefined." You’ll need to adjust the code in database.js and server.js to correctly pass the connection string.

@shr3012, have you created and updated the .env file in the server directory which is necessary to run the backend?

**Comment by shr3012** [comment]
yes, their is an issue after modifying few lines of code it is running
